src/xtgeo/grid3d/grid.py

```diff
@@ -350,13 +382,13 @@ def __init__(
         actnumsv: np.ndarray,
         dualporo: bool = False,
         dualperm: bool = False,
-        subgrids: OrderedDict = None,
+        subgrids: Optional[OrderedDict] = None,
```

Comment:
Question, since python 3.7(?) all dicts are now ordered; should we skip the OrderedDicts()?

Reply:
I asked Chat GPT and it seems to think so 🤖